### PR TITLE
Import Scanner for IBM APP Scan

### PR DESCRIPTION
**Pull request by propersam**
Import Scanner for IBM APP scan file #1075 
When submitting a pull request, please make sure you have completed the following checklist:

- [X] Your code is flake8 compliant 
- [X] Add applicable tests to the unit tests.

### CODE REVIEW

File: tests/ibm_appscan_test.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+from selenium import webdriver
```
Comment (Maffooch) [inline tests/ibm_appscan_test.py]: A selenium is good, but a little slow compared to what could be done. Check out this [parser test](https://github.com/propersam/django-DefectDojo/blob/9280eb868d8eaadba4f6fd02676680393f73cf96/dojo/unittests/test_immuniweb_parser.py).

Comment (propersam) [reply tests/ibm_appscan_test.py]: I wanted to use a parser test... but the sample file of IBM APP Scan is very large

Comment (Maffooch) [reply tests/ibm_appscan_test.py]: Can it be reduced in size?

Comment (propersam) [reply tests/ibm_appscan_test.py]: I don't know how possible that is...

IBMApp Scanner xml result is a really sophisticated xml file by design.

File: tests/ibm_appscan_test.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        Select(driver.find_element_by_id("id_scan_type")).select_by_visible_text("IBM AppScan DAST")
+        # Upload Scan result file
+        scanner_file = os.path.join(dir_path, "ibm_appscan_xml_file.xml")
+        driver.find_element_by_name("file").send_keys(scanner_file)
+        # click on upload button
+        driver.find_element_by_css_selector("input.btn.btn-primary").click()
+        # Query the site to determine if the finding has been added
+        productTxt = driver.find_element_by_tag_name("BODY").text
+        # Assert the query to determine status or failure
+        self.assertTrue(re.search(r'IBM App Scan processed, a total of 27 findings were processed', productTxt))
```
Comment (Maffooch) [inline tests/ibm_appscan_test.py]: Need to update the name here.

File: tests/ibm_appscan_test.py (hunk 1 above)
Comment (Maffooch) [reply tests/ibm_appscan_test.py]: I've checked it out and you're right. There's not much that can be done about this. Great work making a parser from that mess of a file.

Comment (propersam) [reply tests/ibm_appscan_test.py]: yea... thanks

**Pull request closed without merging**